build.sbt

```diff
@@ -10,6 +10,28 @@ val circeVersion = "0.10.0"
 enablePlugins(GitVersioning)
 git.useGitDescribe := true
 
+val artifactoryUrl:java.net.URL = sys.env.get("ARTIFACTORY_URL").map(new java.net.URL(_)).getOrElse(new java.net.URL("http://localhost"))
```

Comment:
Does this localhost thing actually work? Might it be better to have `artifactoryUrl` be an `Option[URL]` and the stuff below accesses it with `artifactoryUrl.get` or is only defined when the option isDefined?

Reply:
No, it doesn't work. I made it an `Option[URL]` 
One of the problem is that I haven't figured out how to stand up an artifactory instance via Docker to test this pr.